src/transformers/modeling_gpt2.py

```diff
@@ -382,14 +382,26 @@ class GPT2DoubleHeadsModelOutput(ModelOutput):
             heads.
     """
 
-    lm_loss: Optional[torch.FloatTensor] = None
+    loss: Optional[torch.FloatTensor] = None
```

Comment:
@sgugger - is that ok with you? For consistency, I would prefer to name all "LMHead" loss and logits, just "loss" and "logits"